parts/windows/kuberneteswindowsfunctions.ps1

```diff
@@ -219,8 +219,8 @@ function Register-NodeLabelSyncScriptTask {
 
     $action = New-ScheduledTaskAction -Execute "powershell.exe" -Argument "-File `"c:\k\windowsnodelabelsync.ps1`""
     $principal = New-ScheduledTaskPrincipal -UserId SYSTEM -LogonType ServiceAccount -RunLevel Highest
-    # trigger this task once(by `-Once) at the time being scheduled(by `-At (Get-Date).Date`) every minute(by `-RepetitionInterval 00:01:00`)
-    $trigger = New-JobTrigger -At  (Get-Date).Date -Once -RepetitionInterval 00:01:00 -RepeatIndefinitely
+    # trigger this task once(by `-Once) at the time being scheduled(by `-At (Get-Date).Date`) every 5 minutes(by `-RepetitionInterval 00:05:00`)
+    $trigger = New-JobTrigger -At  (Get-Date).Date -Once -RepetitionInterval 00:05:00 -RepeatIndefinitely

```

Comment:
May I ask why we need to change it to 5 minutes?

Reply:
Had offline sync with @charliedmcb ,  so that a big cluster won't be pinged too much.